app/src/main/res/layout/fragment_ink_dialog.xml

```diff
@@ -56,35 +58,35 @@
             app:barrierDirection="start"
             app:constraint_referenced_ids="ink_stroke_button, ink_color_container" />
 
-        <androidx.appcompat.widget.AppCompatImageButton
+        <ImageButton
```

Comment:
Just curious Bianca, is AppCompatImageButton worse on accessibility aspects?

Reply:
I made those changes because I actually saw this message in Android Studio:

"This will automatically be used when you use Button/TextView/ImageView/ImageButton in your layouts and the top-level activity / dialog is provided by appcompat. You should only need to manually use this class when writing custom views."

I also realized that if we use the AppCompat components directly in the xml layouts, they do not raise warnings. For all components other than TextView, developers should add contentDescriptions. In case we use the AppCompat version, there is no warning which says that we forgot to add the contentDescription, but for the classic components they appear.

Reply:
oh! didn't know about that! Noce to learn about this! Thanks for the explanation, Bianca! :-)